### PR TITLE
Tweaks outfit datums

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
admin: Added hardsuit outfits for the Captain, HoS, and CMO. Removed old Shaft Miner (Asteroid) outfits. Gave Chief Engineer (Hardsuit) engineering scanner goggles. Added debug outfit for testing.
/:cl:

I added hardsuit outfit datums for Cap/HoS/CMO so admins have an easier time dressing up players. I removed Shaft Miner (Asteroid), since the asteroid has been dead for a long time. 
Also added a "debug" outfit so I don't have to spend 2 1/2 seconds spawning equipment when testing.

### CODE REVIEW

File: code/modules/jobs/job_types/cargo_service.dm [hunk 1]
```diff
@@ -130,14 +125,10 @@ Shaft Miner
 		S.ToggleHood()
 
 /datum/outfit/job/miner/equipped/asteroid
```
Comment (DingleFlop) [inline code/modules/jobs/job_types/cargo_service.dm]: This is still named asteroid

**Comment by Tlaltecuhtli** [comment]
can you increase the number of jumpsuits and backpacks in the department clothing vendors? because there is like only 1 of each for most

**Comment by 81Denton** [comment]
I'll do that in a separate PR, looks like vendor numbers are all over the place.

File: code/modules/jobs/job_types/medical.dm [hunk 2]
```diff
@@ -48,6 +48,14 @@ Chief Medical Officer
 
 	chameleon_extras = list(/obj/item/gun/syringe, /obj/item/stamp/cmo)
 
+/datum/outfit/job/cmo
```
Comment (DingleFlop) [inline code/modules/jobs/job_types/medical.dm]: Overwrites /datum/outfit/job/cmo
change to /datum/outfit/job/cmo/hardsuit

Comment (81Denton) [reply code/modules/jobs/job_types/medical.dm]: Darn, don't drink and PR kids

**Comment by optimumtact** [comment]
why did you buff the CE?

**Comment by 81Denton** [comment]
That outfit already gives him the magboots and insulated gloves, I figured that it might as well include his complete gear with engineering scanner goggles.

**Pull request merged**